src/SampleWebApp/Models/BookDataContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Data
+{
+    public class BookDataContext : DbContext
```

Comment:
Is it expected that scaffolding would generate two separate DbContexts for the two classes?